quickstarts/gradle/quarkus/README.md

```diff
@@ -0,0 +1,82 @@
+
+## Requirements:
+
+- JDK 11+
+- Kubernetes Cluster (Minikube, OpenShift, CRC, etc.)
+
+### For Kubernetes
+Make sure your Minikube instance is running and docker daemon is exposed. If not please run these commands:
+```shell
+$ minikube start
+$ eval $(minikube -p minikube docker-env)
+```
+
+Build the application, and the container image:
+```shell
+$ ./gradlew build k8sResource
+
+> Task :k8sResource
+k8s: Running in Kubernetes mode
+k8s: Running generator quarkus
+k8s: quarkus: Using Docker image quay.io/jkube/jkube-java:0.0.13 as base / builder
+k8s: Using resource templates from /home/anurag/Work/jkube/quickstarts/gradle/quarkus/src/main/jkube
+k8s: jkube-controller: Adding a default Deployment
+k8s: jkube-service: Adding a default service 'quarkus' with ports [8080]
+k8s: jkube-service-discovery: Using first mentioned service port '8080' 
+k8s: jkube-revision-history: Adding revision history limit to 2
+k8s: validating /home/anurag/Work/jkube/quickstarts/gradle/quarkus/build/classes/java/main/META-INF/jkube/kubernetes/quarkus-service.yml resource
+k8s: validating /home/anurag/Work/jkube/quickstarts/gradle/quarkus/build/classes/java/main/META-INF/jkube/kubernetes/quarkus-deployment.yml resource
+
+BUILD SUCCESSFUL in 14s
+7 actionable tasks: 1 executed, 6 up-to-date
+```
+
+Start docker build  by hitting the build task.
+```shell
+$ ./gradlew k8sBuild
+
+> Task :k8sBuild
+k8s: Running in Kubernetes mode
+k8s: Running generator quarkus
+k8s: quarkus: Using Docker image quay.io/jkube/jkube-java:0.0.13 as base / builder
+k8s: Building container image in Kubernetes mode
+k8s: [gradle/quarkus:1.7.0] "quarkus": Created docker-build.tar in 378 milliseconds
+k8s: [gradle/quarkus:1.7.0] "quarkus": Built image sha256:c1df2
+k8s: [gradle/quarkus:1.7.0] "quarkus": Removed old image sha256:04421
+
+BUILD SUCCESSFUL in 10s
+1 actionable task: 1 executed
+
+```
+
+Deploy your application on Kubernetes cluster.
+```shell
+$ ./gradlew k8sApply
+
+> Task :k8sApply
+k8s: Running in Kubernetes mode
+k8s: Running generator quarkus7s]
+k8s: quarkus: Using Docker image quay.io/jkube/jkube-java:0.0.13 as base / builder
+k8s: Using Kubernetes at https://192.168.49.2:8443/ in namespace null with manifest /home/anurag/Work/jkube/quickstarts/gradle/quarkus/build/classes/java/main/META-INF/jkube/kubernetes.yml k8sApply
+k8s: Updating Service from kubernetes.yml
+k8s: Updated Service: build/jkube/applyJson/default/service-quarkus-2.json
+k8s: Creating a Deployment from kubernetes.yml namespace default name quarkus
+k8s: Created Deployment: build/jkube/applyJson/default/deployment-quarkus-2.json
+k8s: HINT: Use the command `kubectl get pods -w` to watch your pods start up
+
+BUILD SUCCESSFUL in 15s
+1 actionable task: 1 executed
+
+$ kubectl get pods                 
+NAME                       READY   STATUS    RESTARTS   AGE
+quarkus-689c7d4bcc-t2jfw   1/1     Running   0          56s
+
+$ minikube service quarkus --url
+http://192.168.49.2:30185
```

Comment:
Could you please add a small section about native mode as well?

Reply:
How to do that?